installer/bundle/bundle_skel.sh

```diff
@@ -801,6 +802,16 @@ case "$installMode" in
                     [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
                 fi
             done
+			echo "----- Installing MySQL ruby gem -----"
+			set +e
+			$OMSAGENT_RUBY_DIR/gem install mysql2 2>/dev/null
+			if [ $? -ne 0 ]; then
+			  echo "MySQL ruby gem was not installed. MySQL ruby plugin would be disabled."
+			  echo "Please check whether libmysqlclient dependency package is installed."
+			  echo "You may need to 'apt-get install libmysqlclient-dev' or 'yum install mysql-devel'."
+			  echo "Then run $OMSAGENT_RUBY_DIR/gem install mysql2 after omsagent installation to enable MySQL ruby plugin."
```

Comment:
I would suggest putting quotes around the command "$OMSAGENT_RUBY_DIR/gem install mysql2" to make it clear exactly what the user has to run.